mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,17 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrderById = (req, res) => {
+  Order.deleteOne({ _id: req.params.orderId })
+    .exec()
+    .then(result => {
+      if (result.deletedCount === 0) {
+        return res.status(404).send({ error: "No order found with that ID" });
+      }
+      res.status(200).send({ message: "Successfully deleted the order" });
```

Comment:
The response object should be JSON, not a simple object. It should be `res.json({ message: "Successfully deleted the order" });` instead of `res.status(200).send({ message: "Successfully deleted the order" });`.